core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3894,12 +3925,16 @@ public synchronized HttpRedirect doQuietDown() {
      *
      * @param block Block until the system really quiets down and no builds are running
      * @param timeout If non-zero, only block up to the specified number of milliseconds
+     * @param reason Quiet reason that will be visible to user
      */
```

Comment:
```suggestion
     * @since TODO
     */
```

(Or perhaps we want this "HTTP API" to be `@Restricted`?)

Reply:
- [ ] Insert todo